DiscordBot/Commands/EventHelper.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DiscordBot.DataAccess;
+using DiscordBot.DataAccess.Exceptions;
+using DiscordBot.DataAccess.Models;
+using DSharpPlus.Entities;
+
+namespace DiscordBot.Commands
+{
+    class EventHelper
+    {
+        private static IEventsSheetsService eventsSheetsService = new EventsSheetsService();
```

Comment:
I wouldn't expect this to have its own instance of the service. I'd expect it to either be passed in to each method or to the constructor for this class 🙂